src/components/Button.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { buttonName, buttonType } = props;
+  return (
+    <button
+      type="button"
+      key={`tecla_${buttonName}`}
+      className={buttonType}
+    >
+      {' '}
+      {buttonName}
+      {' '}
+    </button>
+  );
+};
+
+Button.defaultProps = {
+  buttonName: 'H',
```

Comment:
Hello dev, I'm curious, `buttonName`: 'H', I'm trying to understand where the H is coming from.

I think it should be `butonName: ''`


Kindly explain in detail for the next reviewer to get the full idea, thanks.

Reply:
Thanks for the review.
This is an arbitrary letter I used to test the default value rendering. I needed something to take some space in the button when I was testing the rendering. The specs do not say anything about the default value in this case either. 
Again, I have now corrected this point also as I believe you have a fair point. For submission leaving an 'H' by default does rise confusion.
Thanks again.
Carlos